Source/WebCore/Modules/remoteplayback/RemotePlayback.h

```diff
@@ -42,7 +43,14 @@ class MediaPlaybackTarget;
 class Node;
 class RemotePlaybackAvailabilityCallback;
 
-class RemotePlayback final : public RefCounted<RemotePlayback>, public ActiveDOMObject, public EventTarget {
+class RemotePlayback final
+    : public RefCounted<RemotePlayback>
+#if !RELEASE_LOG_DISABLED
+    , private LoggerHelper
```

Comment:
Nit: Inheriting from LoggerHelper isn't strictly necessary for a non-subclassed class.

Reply:
Good idea, changed